esx_service/cli/vmdkops_admin.py

```diff
@@ -311,8 +311,10 @@ def list_of_values(string):
         given = string.split(',')
         for g in given:
             if g not in allowed:
-                msg = "invalid choice: {0} (choose from {1})".format(g,
-                                                                     allowed)
+                msg = (
+		   'invalid choices: {0} (choices must be a comma separated list of '
+                   'only the following words {1}. '  
+		   'No spaces are allowed between choices.)').format(g, allowed)
```

Comment:
This prints list with spaces. Should we remove space from list https://github.com/vmware/docker-volume-vsphere/blob/master/esx_service/cli/vmdkops_admin.py#L119